### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by adamdruppe**
assorted but fixes from the call

### CODE REVIEW

File: app/controllers/enrollments_controller.rb [hunk 1]
```diff
@@ -140,7 +143,7 @@ def enrollment_submitted_crm_actions
       cm.Apply_Button_Enabled__c = false
 
       cm.Middle_Name__c = @enrollment.middle_name
-      #cm.Phone = @enrollment.phone
+      # cm.Phone = @enrollment.phone
```
Comment (sadleb) [inline app/controllers/enrollments_controller.rb]: what's up with this change?

File: app/controllers/enrollments_controller.rb [hunk 2]
```diff
@@ -184,6 +187,7 @@ def enrollment_submitted_crm_actions
       task.WhoId = contact.Id
       task.OwnerId = contact.OwnerId
       task.IsReminderSet = false
```
Comment (sadleb) [inline app/controllers/enrollments_controller.rb]: Actually, we probably do want the reminder set.  Any reason you explicitly set this false?

File: app/views/users/_areas.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-    <% List.find_by_friendly_name('bz_regions').items.each do |region, idx| %>
+    <% List.find_by_friendly_name('bz_regions').items.each_with_index do |region, idx| %>
```
Comment (sadleb) [inline app/views/users/_areas.html.erb]: I still don't get what was going on here, but if this works, great!

Comment (adamdruppe) [inline app/views/users/_areas.html.erb]: On Thu, Mar 12, 2015 at 08:23:56AM -0700, Brian Sadler wrote:
> I still don't get what was going on here, but if this works, great!

The .each function doesn't give the index which I need to check
if it was in the array. The correct function was .each_with_index -
without it, that value was nil, which is where the error was coming
from.

File: app/controllers/enrollments_controller.rb (hunk 2 above)
Comment (adamdruppe) [inline app/controllers/enrollments_controller.rb]: On Thu, Mar 12, 2015 at 08:22:55AM -0700, Brian Sadler wrote:
> Actually, we probably do want the reminder set.  Any reason you explicitly set this false?

It is a required field and I wasn't sure what to set so I just did
false.

But True probably makes more sense so i;ll do that.

**Comment by sadleb** [comment]
looks good1

**Pull request merged**